src/sage/quadratic_forms/quadratic_form__theta.py

```diff
@@ -198,16 +197,16 @@ def theta_by_cholesky(self, q_prec):
     x = [0] * n
 
     # 2. Compute bounds
-    #Z = sqrt(T[i] / Q[i,i])      # IMPORTANT NOTE: sqrt preserves the precision of the real number it's given... which is not so good... =|
-    #L[i] = floor(Z - U[i])       # Note: This is a Sage Integer
-    #x[i] = ceil(-Z - U[i]) - 1   # Note: This is a Sage Integer too
+    # Z = sqrt(T[i] / Q[i,i])      # IMPORTANT NOTE: sqrt preserves the precision of the real number it's given... which is not so good... =|
+    # L[i] = floor(Z - U[i])       # Note: This is a Sage Integer
+    # x[i] = ceil(-Z - U[i]) - 1   # Note: This is a Sage Integer too
 
     done_flag = False
     from_step4_flag = False
     from_step3_flag = True        # We start by pretending this, since then we get to run through 2 and 3a once. =)
 
-    #double Q_val_double;
-    #unsigned long Q_val;                 // WARNING: Still need a good way of checking overflow for this value...
+    # double Q_val_double;
```

Comment:
these two variables are defined line 245. So these 2 lines can be removed I think.